.gitignore

```diff
@@ -77,6 +77,7 @@ sorc/gfs_post.fd
 sorc/gfs_wafs.fd
 sorc/gldas.fd
 sorc/gsi.fd
+sorc/gsi_utils.fd
```

Comment:
Lines 105 and 106 reference `global_enkf.fd` and `global_gsi.fd`.   These directories are not created by the updated `link_workflow.sh`.   Instead we have `gsi.fd` and `enkf.fd`.     We can remove `global_gsi.fd` and `global_enkf.fd`.   `gsi.fd` is already in `.gitignore`.  We can add `enkf.fd`.   

For the time being `link_workflow.sh` creates `ncdiag.fd`.   For completeness we could replace `ncdiag_cat.fd` in `.gitignore` with `ncdiag.fd`

Reply:
That raises an interesting question.
`gsi.fd` from `checkout.sh` is the "umbrella" GSI containing the source codes for `GSI`, `EnKF` and soon to be removed `ncdiag`.
Do (Should) we need to "link" `GSI/src/gsi` and `GSI/src/enkf` to `sorc`?
EE2 standards say yes.
What if we call the `gsi.fd` from `checkout.sh` as `GSIApp.fd` and then retain the symlinks?
Just a thought for now and can be worked later.
I will pin this for discussion w/ NCO.